app/controllers/application_letters_controller.rb

```diff
@@ -36,8 +36,8 @@ def create
 
   # PATCH/PUT /applications/1
   def update
-    if @application_letter.update(application_params)
-      redirect_to @application_letter, notice: 'Application was successfully updated.'
+    if @application_letter.update_attributes(application_params)
+      redirect_to :back, notice: 'Application was successfully updated.'
```

Comment:
It redirects back to the current site. If a organizer visits the event overview page, he/she can mark many applications as accepted/rejected without going back to the overview page every time. The HTTP referred is only neccessary for the test.